fontforgeexe/charview.c

```diff
@@ -1495,103 +1495,122 @@ void CVDrawSplineSet(CharView *cv, GWindow pixmap, SplinePointList *set,
 }
 
 
-void CVDrawSplineSetOutlineOnly(CharView *cv, GWindow pixmap, SplinePointList *set,
-				Color fg, int dopoints, DRect *clip, enum outlinesfm_flags strokeFillMode ) {
+void CVDrawSplinePointList(CharView *cv, GWindow pixmap, SplinePointList *set, Color fg,
+                         enum outlinesfm_flags strokeFillMode, float xoff, float yoff, real scale)
+{
     SplinePointList *spl;
-    int activelayer = CVLayer(&cv->b);
-    CharViewTab* tab = CVGetActiveTab(cv);
-
-    if( strokeFillMode == sfm_fill ) {
-    	GDrawFillRuleSetWinding(pixmap);
-    }
-
-    for ( spl = set; spl!=NULL; spl = spl->next ) {
+    int height = 0;
+    for (spl = set; spl != NULL; spl = spl->next)
+    {
+        Color fc = spl->is_clip_path ? clippathcol : fg;
```

Comment:
I haven't tried this, but from the code it seems like you turn on the metrics option and you get the outline in the foreground color? Is that always going to make sense? What if you have a mix of closed and open contours (maybe you're in the middle of developing a stroked font) and want to see the contrast?